src/app/hearing-part/components/hearing-search-table/hearing-search-table.component.ts

```diff
@@ -21,6 +21,12 @@ import { Status } from '../../../core/reference/models/status.model';
 })
 export class HearingSearchTableComponent implements OnInit, OnChanges {
     @ViewChild(MatPaginator) paginator: MatPaginator;
+
+    @Input() set pagingValues(pagingValues: PageEvent) {
```

Comment:
To inject previous values I decided to go with a new input. Please @zielinskikamil @r-kruk-kainos give me your opinion on that. I could use another service to syncrhonise the three components (HearingSearchComponent, HearingFilterComponent and HearingSearchTableComponent) but I found it an overkill. Using child component reference (via @ViewChild annotation) to set its properties is not perfect either because it doesn't trigger the change dectection mechanism with ChangeDetectionStrategy.OnPush. I'd have to call the changedetection manually which I wanted to avoid. Tell me what you think